lib/sidebar.js

```diff
@@ -21,12 +21,10 @@ define(function () {
     sidebar.appendChild(container);
 
     self.getWidth = function getWidth() {
-      if (sidebar.classList.contains('hidden')) {
+      if (sidebar.classList.contains('hidden') || sidebar.firstElementChild.offsetHeight === 0) {
```

Comment:
`sidebar.firstElementChild` wound be the `button` variable, firstElementChild is not really spezific ...

second, should it not be a `&&` ?